decidim-elections/lib/decidim/votings/seeds.rb

```diff
@@ -6,8 +6,9 @@ module Decidim
   module Votings
     class Seeds < Decidim::Seeds
       def call
-        3.times do |_n|
-          voting = create_voting!
+        3.times do |n|
```

Comment:
@andreslucena  Why don't we create one record for voting type ? I am thinging that we could  use 

Decidim::Votings::Voting.voting_types.each instead of 3.times...

Reply:
Done in [aa7cdf4](https://github.com/decidim/decidim/pull/12152/commits/aa7cdf426ca710d5d5066a954e6edd5e6e5be43e)